plasma/root_chain/deployer.py

```diff
@@ -37,10 +37,10 @@ def compile_contract(self, path, args=()):
         contract_file = open("contract_data/%s.json" % (file_name.split('.')[0]), "w+")
         json.dump(abi, contract_file)
         contract_file.close()
-        return abi, bytecode
+        return abi, bytecode, contract_name
 
     def create_contract(self, path, args=(), gas=4410000, sender=t.k0):
-        abi, bytecode = self.compile_contract(path, args)
+        abi, bytecode, contract_name = self.compile_contract(path, args)
```

Comment:
This change needs to be implemented here:
https://github.com/omisego/plasma-mvp/blob/30b7cfbd9e713f3093cda1cb2660f518fe523f0c/tests/conftest.py#L60 

Can be implemented as
```py
abi, bytecode, _ = Deployer().compile_contract(path, args)
```
as we don't need the contract name in order to complete that test.

`make test` fails otherwise.